data/data/agent/systemd/units/load-config-iso@.service.template

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=If a config ISO is mounted, copy its files to the system
 Wants=network-online.target
-After=network-online.target
+After=network-online.target agent-auth-token-status.service
+Requires=agent-auth-token-status.service
```

Comment:
I don't know if this dependency is needed as this service shouldn't be running at day 2.